README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
@vancem Does the following work for you?

---

### Developer information

This repository uses [AppVeyor](https://www.appveyor.com/) to automatically build and test pull requests, which allows
the community to easily view build results. The build status reflected here is the AppVeyor build status of the
**master** branch.

[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)

> :warning: Builds produced by AppVeyor are not considered official builds of PerfView, and are not signed or otherwise validated for safety or security in any way. This build integration is provided as a convenience for community participants, but is not endorsed by Microsoft nor is it considered an official release channel in any way. For information about official builds, see [Getting PerfView]() below.